static/sass/styles.scss

```
@@ -846,3 +846,9 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
   background-color: $colors--dark-theme--background-default;
   color: $colors--dark-theme--text-default;
 }
+
+.p-highlight--vertical {
+  margin-left: 0;
+  padding-left: 1rem;
+  @include vf-highlight-bar(#E95420, left);
+}
```

```suggestion
.p-highlight--vertical {
  @include vf-highlight-bar(#E95420, left);
  
  margin-left: 0;
  padding-left: 1rem;
}
```
We usually put these at the top